xcode/Subconscious/Shared/Components/Feed.swift

```diff
@@ -25,7 +25,10 @@ struct FeedNavigationView: View {
         DetailStackView(app: app, store: detailStack) {
             ScrollView {
                 LazyVStack(spacing: 0) {
-                    if let feed = store.state.entries {
+                    switch (store.state.status, store.state.entries) {
```

Comment:
Introduced an explicit `status` for the feed instead of checking whether `entries` was `nil`. This lets us be smarter about when to show the loading vs empty state.